api/v2/swagger.json - Schema for organization is missing _Log_ link

build **Nightly**
installed from: **quay.io/influxdb/influx:nightly** 
buildtime: **2019-02-06T07:09:06**
related https://github.com/influxdata/influxdb/issues/10677

```
ts=2019-02-06T09:59:13.967440Z lvl=info msg="Welcome to InfluxDB" log_id=0DSZniBl000 version=nightly commit=cd6fb3cd6 build_date=2019-02-06T07:09:06Z
```
*Use Case*

I'm creating some Jackson - Json mappings to Java beans for working with api/v2.  

I'm using `http://localhost:9999/api/v2/swagger.json` schema as a guide to bean structure.  

When checking the mapping for Organization, the following error was thrown. 
```
javax.ws.rs.ProcessingException: Error reading entity from input stream.
	at com.bonitoo.qa.TestRunner.getOrgId(TestRunner.java:185)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: 
**Unrecognized field "log"** (class com.bonitoo.qa.flux.rest.artifacts.OrganizationLinks), not marked as ignorable (7 known properties: "secrets", "buckets", "self", "tasks", "dashboards", "members", "labels"])
 at [Source: (org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream); line: 1, column: 183] (through reference chain: com.bonitoo.qa.flux.rest.artifacts.OrganizationArray["orgs"]->java.util.ArrayList[0]->com.bonitoo.qa.flux.rest.artifacts.Organization["links"]->com.bonitoo.qa.flux.rest.artifacts.OrganizationLinks["log"])
	at com.bonitoo.qa.TestRunner.getOrgId(TestRunner.java:185)

```
After adding the field `log` to the bean, deserialization worked successfully. 

*Expected Behaviour*
That the `swagger.json` document should be up-to-date.  That schemas should reflect the data structures provided or consumed by the Influx2 V2 api. 

*Snippet*
from current swagger.json schema
```
...
Organization: {
properties: {
id: {
readOnly: true,
type: "string"
},
links: {
example: {
buckets: "/api/v2/buckets?org=myorg",
dashboards: "/api/v2/dashboards?org=myorg",
labels: "/api/v2/orgs/1/labels",
members: "/api/v2/orgs/1/members",
secrets: "/api/v2/orgs/1/secrets",
self: "/api/v2/orgs/1",
tasks: "/api/v2/tasks?org=myorg"
},
properties: {
buckets: {
format: "uri",
readOnly: true,
type: "string"
},
dashboards: {
format: "uri",
readOnly: true,
type: "string"
},
labels: {
format: "uri",
readOnly: true,
type: "string"
},
members: {
format: "uri",
readOnly: true,
type: "string"
},
secrets: {
format: "uri",
readOnly: true,
type: "string"
},
self: {
format: "uri",
readOnly: true,
type: "string"
},
tasks: {
format: "uri",
readOnly: true,
type: "string"
}
},
readOnly: true,
type: "object"
},
name: {
type: "string"
},
owners: {
$ref: "#/components/schemas/Owners"
},
status: {
default: "active",
description: "if inactive the organization is inactive.",
enum: [
"active",
"inactive"
],
type: "string"
}
},
required: [
"name"
]
},
```

--- a/http/swagger.yml
+++ b/http/swagger.yml
@@ -5593,6 +5593,7 @@ components:
             buckets: "/api/v2/buckets?org=myorg"
             tasks: "/api/v2/tasks?org=myorg"
             dashboards: "/api/v2/dashboards?org=myorg"
+            logs: "/api/v2/orgs/1/logs"
           properties:
             self:
               $ref: "#/components/schemas/Link"
@@ -5609,6 +5610,8 @@ components:
             tasks:
               $ref: "#/components/schemas/Link"
             dashboards:
+              $ref: "#/components/schemas/Link"
+            logs:
               $ref: "#/components/schemas/Link"
         id:
           readOnly: true
@@ -6715,6 +6718,7 @@ components:
                 cells: "/api/v2/dashboards/1/cells"
                 owners: "/api/v2/dashboards/1/owners"
                 members: "/api/v2/dashboards/1/members"
+                logs: "/api/v2/dashboards/1/logs"
               properties:
                 self:
                   $ref: "#/components/schemas/Link"
@@ -6723,6 +6727,8 @@ components:
                 members:
                   $ref: "#/components/schemas/Link"
                 owners:
+                  $ref: "#/components/schemas/Link"
+                logs:
                   $ref: "#/components/schemas/Link"
             id:
               readOnly: true

